src/test/rgw/test_rgw_iam_policy.cc

```diff
@@ -785,7 +785,9 @@ class ManagedPolicyTest : public ::testing::Test {
 protected:
   intrusive_ptr<CephContext> cct;
 public:
-  ManagedPolicyTest() : cct(new CephContext(CEPH_ENTITY_TYPE_CLIENT)) {}
+  ManagedPolicyTest() {
+    cct.reset(new CephContext(CEPH_ENTITY_TYPE_CLIENT), false);
+  }
```

Comment:
nit: could just add `, false` to the constructor call above

Reply:
https://github.com/ceph/ceph/blob/7e378c14e98e046a62115b2c43f7045b38a98b4e/src/test/rgw/test_rgw_iam_policy.cc#L222

https://github.com/ceph/ceph/blob/7e378c14e98e046a62115b2c43f7045b38a98b4e/src/test/rgw/test_rgw_iam_policy.cc#L1051

Keep code uniformity here.

Reply:
Hi @cbodley, QA needed? Otherwise I will merge this one.